from test_scribe.error import Error


def raise_custom_error():
    raise Error("foo")
